Cdnfsd_TransparentCDN_Engine.php

```diff
@@ -81,10 +81,13 @@ public function purge( $urls ) {
 		$this->_get_token();
 
 		$invalidation_urls = array();
-
+		//Included a regex filter because some of our clients reported receiving urls as "True" or "False"
+		$url_pattern = "/https?:\/\/(www\.)?[-a-zA-Z0-9]{1,256}\.?[a-zA-Z0-9]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/";
```

Comment:
The PCRE regex has an issue (Unknown modifier '/').  Review these tests:

```
$ php -r '$url = "https://www.exmaple.com/test-page?queryarg=123"; $url_pattern = "/https?:\/\/(www\.)?[-a-zA-Z0-9]{1,256}\.?[a-zA-Z0-9]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/"; var_dump( preg_match( $url_pattern, $url ) );'
Warning: preg_match(): Unknown modifier '/' in Command line code on line 1
bool(false)
```

When removing the two slashes in the pattern:
```
$ php -r '$url = "https://www.exmaple.com/test-page?queryarg=123"; $url_pattern = "/https?:\/\/(www\.)?[-a-zA-Z0-9]{1,256}\.?[a-zA-Z0-9]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&=]*)/"; var_dump( preg_match( $url_pattern, $url ) );'
int(1)
```

If you intended to have the slashes in the character class, then you need only one and need to either escape it or change your regex delimiter.